src/sage/graphs/digraph_generators.py

```diff
@@ -841,12 +846,24 @@ def Circulant(self, n, integers):
           that there is an edge from `i` to `j` if and only if `(j-i) \pmod{n}`
           is an integer
 
-        EXAMPLES::
+        EXAMPLES:
 
-            sage: digraphs.Circulant(13,[3,5,7])
-            Circulant graph ([3, 5, 7]): Digraph on 13 vertices
+        Construct and show the circulant graph [3, 5, 7], a digraph on 13
+        vertices::
 
-        TESTS::
+            sage: g = digraphs.Circulant(13, [3, 5, 7])
+            sage: g.show()                          # long time                             # needs sage.plot
+
+        The Koh-Tindell digraph [LM2024]_ is the circulant digraph of order 7
+        with parameters `[1, 5]`.  This `2`-diregular digraph is is
```

Comment:
`is is` -> `is`

Reply:
Ha. I do check. I usually run the command `sudo sage -b && sudo sage --docbuild reference html` after the changes made and before pushing and check for the documentation. This one was left mistakenly.